package.json

```diff
@@ -97,8 +98,8 @@
     "@radix-ui/react-tooltip": "^1.0.7",
     "class-variance-authority": "^0.7.0",
     "clsx": "^2.1.0",
-    "date-fns": "^4.0.0",
-    "react-day-picker": "^9.0.8",
+    "date-fns": "^3.3.1",
```

Comment:
Downgrading react-day-picker as version 9 contained breaking changes that cause the build to error (and the cal to not function correctly)